### PR TITLE
feat: start server via npx

### PR DESCRIPTION
**Pull request by lichangwei**
add a simple way to start redis memory server

### CODE REVIEW

**Comment by mhassan1** [comment]
thanks for the contribution!

this is an interesting idea. what is the use case for this?

**Comment by shimondoodkin** [comment]
I can imagine the use case for it could be for tests in a not installed environment liked docker. 

maybe to use with not javascript tests.

for example to run it as a background process to have a server.
 
RUN npx cross-env REDISMS_PORT=6379 npx redis-memory-server & 
RUN run c++ tests..

**Comment by mhassan1** [comment]
That makes sense. @lichangwei, there is some outstanding feedback.

**Review by mhassan1 — CHANGES_REQUESTED** [review]

**Comment by mhassan1** [comment]
@lichangwei It looks like I never submitted the feedback I wrote over 2 years ago... Sorry about that! It's submitted now.

File: package.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "name": "redis-memory-server",
   "version": "0.3.1",
   "description": "Redis Server for testing. The server will allow you to connect your favorite client library to the Redis Server and run parallel integration tests isolated from each other.",
+  "bin": "bin/index.js",
```
[the remark this answers is not in the document]
Comment (lichangwei) [reply package.json:5]: fixed

File: README.md [hunk 2]
```diff
@@ -60,6 +60,14 @@ or via [`package.json`](#options-which-can-be-set-via-packagejson).
 
 ## Usage
 
+### Start server via npx
```
[the remark this answers is not in the document]
Comment (lichangwei) [reply README.md]: fixed.

File: bin/index.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    instance: {
+      port: process.env.REDISMS_PORT ? parseInt(process.env.REDISMS_PORT, 10) : undefined,
+    },
+  });
+
+  const host = await server.getHost();
+  const port = await server.getPort();
+
+  console.log(`Redis server running at: ${host}:${port}`);
+})();
```
[the remark this answers is not in the document]
Comment (lichangwei) [reply bin/index.js:21]: fixed

**Review by mhassan1 — APPROVED** [review]

**Comment by mhassan1** [comment]
thank you for the contribution, @lichangwei!

**Pull request merged**